scripts/reana

```diff
@@ -104,6 +106,91 @@ down () {
     kubectl delete -Rf configuration-manifests
 }
 
+demo () {
+    # Used to run REANA demo workflows.
+    # For now, hardcoded to run only helloworld-workflow.
+    demonstration=$2
+
+    # The code to check for flags is from http://stackoverflow.com/q/2875424
+    # Not sure if this is a good way or not.
+    flag_verbose=$(echo "$*" | grep -e "--verbose" -o)
+    flag_devel=$(echo "$*" | grep -e "--devel" -o)
+
+    if [ "$demonstration" != "" ]; then
+        #Check that there exists such a demo as inputted by user.
+        if echo "$reana_demos" | grep -qE "^$demonstration$"
+            then
+            # FIXME: Check REANA cluster is accessible. Simple ping/pong, or interaction with kubectl?
+
+            # Get URL of reana-workflow-controller
+            controller_url=$(reana get reana-workflow-monitor)
+
+            # Submit a helloworld-workflow job to workflow-controller
+            workflow_id=$(curl --fail --silent \
+                            -H "Content-Type: application/json" \
+                            -X POST -d \
+                            "{\"experiment\":\"atlas\",
+                              \"toplevel\":\"github:diegodelemos/reana-demo-helloworld@2-add-helloworld-workflow\",
```

Comment:
see reanahub/reana-demo-helloworld#4. Once it is merged `\"toplevel\":\"github:diegodelemos/reana-demo-helloworld@2-add-helloworld-workflow\"` should be changed for `\"toplevel\":\"github:reanahub/reana-demo-helloworld\"`.